### DISCUSSION TITLE
A static or shared library for MXMRT 1062

### DISCUSSION THREAD

**Discussion by 8888clockradio** [micropython/micropython#10352] [m.iMX RT / Teensy 4.x]
Is there an easy way to just get the damned thing to work as a library on Teensy 4.1? I could potentially port the unix one over too as gcc-none-eabi is getting pretty advanced it does gnu++20 code, and can do threading if you set a couple flags. I've been struggling with this forever, and I just want it to work as a library that can tack on to an other core.

**Comment by 8888clockradio** [comment]
These ports are kind of all off base. Everyones using the wrong processor (ESP) like idiots. NXP makes the audio hardware, dudes. I want vocoders and wavetable synths and stuff. Best bet for me is just port the teensy audio library and take advantage of fspmath in the NXP libraries when porting it to c python. Unless you got an easy static or shared library option. If I could just get cpython to compile on arm, I could just use flipping cython or something. Micropython isn't necessary very likely, but I may just port teensy audio library. Python just got to be a bitch on the 32-bit memory range ARMs.

**Comment by jimmo** [comment]
@8888clockradio I've deleted your other comments on this thread as they are not productive or in line with the code of conduct. Please see https://github.com/micropython/micropython/blob/master/CODEOFCONDUCT.md
